src/components/SheetBuilder/SheetBuilderForm/ConfirmButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type Props = {
+  confirm(): void;
+};
+
+const ConfirmButton = ({ confirm }: Props) => (
+  <button
+    onClick={() => confirm()}
```

Comment:
Isso aqui pode ser simplificado pra:
```suggestion
    onClick={ confirm }
```